web/styles/settings.css

```diff
@@ -633,6 +633,12 @@ input[type="checkbox"] {
             font-weight: bold;
             word-break: break-all;
         }
+
+        .zulip-icon {
```

Comment:
As above, this is probably too generic a selector. I'm also not understanding where the `font-size` calculation is coming from. As above, you're using positioning here, but your offset is expressed in pixels, not ems.

Reply:
I have changed the font-size to 1em (16px) equal to font size of 'all" .

Reply:
I don't understand what that accomplishes, and the other feedback I've left has not been addressed.

Reply:
I've updated the font size of '@' before 'all' in /#settings/notifications. Since the font size of 'all' is 16px (1em), I set the font size of '@' to 1em as well.

My earlier calculation was based on observation. I also reviewed the font sizes of other icons, which initially influenced my approach.

Reply:
I don't think it's necessary to set `font-size: 1em`, as that's the size that will be inherited anyway.

Reply:
okh, removed the font-size.